Extension/src/LanguageServer/configurations.ts

```diff
@@ -1998,7 +2006,6 @@ export class CppProperties {
             }
 
             // Escape the path string for literal use in a regular expression
-            // Need to escape any quotes to match the original text
             let escapedPath: string = curPath.replace(/"/g, '\\"');
```

Comment:
What about this case?

Reply:
That's the one I dismissed

Reply:
oh, but I didn't mean to remove the comment. That was left over from my poking around at it.